### ISSUE TITLE
Expose superscript and subscript text

### ISSUE THREAD

**Issue by aardrian** [FreedomScientific/VFO-standards-support#668]
### Summary

Subscript and superscript text is not exposed. An example:

1. Go to https://cdpn.io/pen/debug/aGgBNZ
2. Read the content using any means.
3. Observe the sub- and super-script is not exposed in any way.
4. Look for a JAWS setting to expose it.
5. Find none.

### Expected result

I expect to hear the sub- and super-script text as marked up by `<sub>` and `<sup>`.

Failing that, I expect there is at least a setting to allow JAWS to expose superscript and subscript text to users.

### Actual result

The text was read without denoting if it is sub- or super-script.

In the _Mixed_ example, the numbers are concatenated and `<sup>14</sup><sub>6</sub>` are announced as "one forty-six".

### Example

- Debug pen: https://cdpn.io/pen/debug/aGgBNZ
- Editable pen: https://codepen.io/aardrian/pen/aGgBNZ

Example code that concatenates values:
```
<sup class="texttop">14</sup><sub>6</sub>C<sup class="texttop">2+</sup><sub>8</sub>
```

### Additional Information

- NVDA has a setting (under Document Formatting) to announce sub- and super-script.
- As of the start of 2019 JAWS had no support for `<sup>` nor `<sup>` (per https://freedomscientific.github.io/VFO-standards-support/html.html) so assuming that is still the case.

#### JAWS version and build number
- JAWS 2022.2202.20
 
#### Operating System and version
- Windows 10
- Windows 11

#### Browser and version:
- Chrome 105
- Firefox 105

**Comment by JAWS-test** [comment]
Older JAWS versions (e.g. 15, 18 and 2019) still output the elements as superscript and subscript, but the current version unfortunately no longer does so

**Comment by jducrot** [comment]
I recently had to do some research on this topic and a blind colleague pointed out that in JAWS 21 (not sure about 22) you have to edit a sound scheme to enable superscript and subscript announcements. It's buried deep down in the settings and nothing comes up when you search the JAWS documentation for the obvious key words. Here are the steps to get there from the JAWS docs:
1. With JAWS running, press `Insert+F2`, then press `S` until Settings Center is selected, and press `Enter`.
2. Press `Ctrl+Shift+D` to load the Default settings configuration.
3. Type “scheme” in the search field, and Tab to the list of choices.
4. Down Arrow until “Active Speech and Sounds Scheme” is chosen. Pressing Space while this option is selected will cycle through the available schemes.
5. Continue pressing Down Arrow until “Modify Schemes…” is chosen. Pressing Space here will bring up a list of schemes which can be modified. (It’s okay to choose this, so long as you don’t save any changes you make to pre-existing schemes.)
6. When the list of modifiable schemes comes up, Tab to `Edit Selected Scheme`, and press Enter.
7. Press `Ctrl+Tab` until the Attributes Page in the multi-tab dialog is shown. Both subscript and superscript are available as configurable attributes in this list.

**Comment by aardrian** [comment]
@jducrot I am using JAWS 2022 and I can get to the screen you describe. I made a new scheme to play a sound for sub- and super-script text but am having no luck with it. As in, the audio file I chose is not played when I encounter sub/super text.

![Edit Scheme dialog showing subscript is selected and has the behavior of 'Play sound' with the audio file descend.wave chosen; Superscript is set up similarly with ascend.wav.](https://user-images.githubusercontent.com/1376607/191057680-f8d60667-0110-4233-92c5-dadde9e13984.png)

I did not mean to make this into a help desk request. However, this does seem a bit buried.

**Comment by Wildebrew** [comment]
There is a common use for these on the web, that is to reference footnotes. Without support for the superscript element, Jaws ends up communicating odd information.
The following HTML has been found on multiple banking websites:
Must be at least 18<sup1</sup> to apply.
...
The superscript reference points to a footnote.
Jaws (tested in 2022) reads this as "must be 181 to apply".
Given how frequent superscript references are on the web, Jaws should have super/subscript announcements as part of the verbosity setting for the browser, ideally they should be announced always in beginner verbosity mode.

**Comment by JAWS-test** [comment]
No matter what I set under Active Speech and Sounds Scheme: JAWS does not output sub and sup or the ARIA roles subscript and superscript in any way.

**Comment by aardrian** [comment]
Still an issue in JAWS 2024. I have confirmed that the audio files I have chosen in my custom scheme are real audio files and play real audio.

![Edit Scheme dialog showing Superscript is selected and has the behavior of 'Play sound' with the audio file ascend.wav chosen; Subscript is set up similarly with descend.wave.](https://github.com/FreedomScientific/standards-support/assets/1376607/63dae238-2def-4bcb-9188-a25a4a6f77d2)

**Comment by SuzanneTaylor** [comment]
This is important for some inline math as well. 

Not only is the beginning of subscript or superscript important, but so is the end. Some places where this is done well:

- With NVDA, MathPlayer, and FF or Chrome, inside of mathml, you hear "sub" as well as "end sub". (JAWS's  MathML reading doesn't mention the end by default)
- With Chrome and NVDA, outside of MathML in ordinary HTML, you hear "subscript" before the subscript content and "baseline" at the end. (if you activated a setting for subscripts and superscripts under document formatting)

I also could not get Scheme adjustments to make a difference, though the UI for it seems VERY robust!

**Comment by AmruthRaj-David** [comment]
Thanks all for the contributors.

Still an issue in JAWS 2025. JAWS doesn't announce the associated Superscript text info.

Here is the Environment which I have tested.

**JAWS Version:** 2025.2412.50

- **Chrome Version:** 136.0.7103.49 (Official Build) (64-bit)
- **Mozilla Firefox Version:** 128.6

**Comment by Wildebrew** [comment]
Given how well Jaws has started handling the other text level semantics like mark or insert or delete, I am surprised this change has not been  made yet.
It's not like people use this markup willy nilly across webpages; pretty much only for math and for footnote references.
